test/collectionUtils/ModifyExtensionTypeTransformer.test.ts

```diff
@@ -3,87 +3,40 @@ import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
 import DataType from "../../src/model/dataTypes/DataType";
 import ModifyExtensionTypeTransformer from "../../src/collectionUtils/ModifyExtensionTypeTransformer";
 import EntityImports from "../../src/model/dataTypes/EntityImports";
-import MemberVariable from "../../src/model/dataTypes/MemberVariable";
-import SystemString from "../../src/model/dataTypes/system/SystemString";
+import FilePath from "../../src/model/dataTypes/FilePath";
 
 describe("ModifyExtensionTypeTransformer", () => {
-  const transformer = new ModifyExtensionTypeTransformer();
+  const path = FilePath.getInstance("/tmp");
+  const transformer = new ModifyExtensionTypeTransformer(path);
   let entityBuilder: EntityDefinitionBuilder;
   let otherEntity: EntityDefinition;
   let extensionEntity: EntityDefinition;
-  let elementType: DataType;
   let extensionType: DataType;
-  let typeType: DataType;
-  let member1: MemberVariable;
-  let member2: MemberVariable;
-  let valueMember: MemberVariable;
 
   beforeEach(() => {
-    elementType = DataType.getInstance("FHIR", "Element", "/tmp");
     extensionType = DataType.getInstance("FHIR", "Extension", "/tmp");
-    typeType = DataType.getInstance("FHIR", "Type", "/tmp");
 
     entityBuilder = new EntityDefinitionBuilder();
     otherEntity = entityBuilder.buildEntityDefinition();
 
-    valueMember = new MemberVariable(
-      typeType,
-      "value",
-      false,
-      undefined,
-      false,
-      [elementType, extensionType]
-    );
-
     entityBuilder.dataType = extensionType;
-    entityBuilder.imports = new EntityImports([elementType]);
-    entityBuilder.memberVariables.push(valueMember);
+    entityBuilder.imports = new EntityImports([]);
 
     extensionEntity = entityBuilder.buildEntityDefinition();
-    [member1, member2] = extensionEntity.memberVariables;
   });
 
   it("should leave non-Extension types alone", () => {
     const result = transformer.transform(otherEntity);
     expect(result).toBe(otherEntity);
-    expect(result.parentDataType).not.toBeNull();
+    expect(result.imports.dataTypes).toBeArrayOfSize(2);
+    expect(result.imports.dataTypes[0].normalizedName).toBe("memberTypeName1");
+    expect(result.imports.dataTypes[1].normalizedName).toBe("memberTypeName2");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)